onmt/opts.py

```diff
@@ -557,12 +523,6 @@ def train_opts(parser):
               help="Step for moving average. "
                    "Default is every update, "
                    "if -average_decay is set.")
-    group.add("--src_noise", "-src_noise", type=str, nargs='+',
```

Comment:
Since all these opts are now in all the dedicated transform files, I think it would be nice to have a specific section of the doc, that would be automatically built from the transforms folder, and show the transform purpose (should be in the class header comment) as well as the specific opts.